src/cockpit/389-console/src/monitor.jsx

```diff
@@ -11,6 +11,7 @@ import AuditLogMonitor from "./lib/monitor/auditlog.jsx";
 import AuditFailLogMonitor from "./lib/monitor/auditfaillog.jsx";
 import ErrorLogMonitor from "./lib/monitor/errorlog.jsx";
 import SecurityLogMonitor from "./lib/monitor/securitylog.jsx";
+import ReplLogAnalysis from "./lib/monitor/replLogAnalysis.jsx";
```

Comment:
Is it possible to import this only when repl-tools are available on the system? For example, on RHEL we don't have the required dependencies for repl-tools, so this tab would be not fully functional and misleading to the users.

Reply:
Good point! I'll disable HTML and PNG options in case the package is missing. Patternfly Charts and CSV should work without it.